plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobCard.qml

```diff
@@ -206,7 +206,11 @@ Item
         onClicked: enabled ? contextMenu.switchPopupState() : {}
         visible:
         {
-            if (!printJob)
+            if(!printJob)
```

Comment:
```suggestion
            if (!printJob)
```

Reply:
This is not in the code style guidelines and I don't really agree with the change.